files/es/web/api/batterymanager/level/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: BatteryManager.level
+slug: Web/API/BatteryManager/level
+tags:
+  - API
+  - Battery API
+  - NeedsMarkupWork
+  - Property
+  - Reference
+translation_of: Web/API/BatteryManager/level
+browser-compat: api.BatteryManager.level
+---
+{{deprecated_header}}{{APIRef("Battery API")}}
+
+Indica el valor actual del nivel de carga, estos valores están entre `0.0` a `1.0`.
+
+## Sintaxis
+
+```js
+var level = battery.level;
+```
+
+`Level` es un número que representa el nivel de carga de la batería del sistema en una escala entre 0.0 y 1.0. Un valor de 0 significa que la `battery`, que es un objeto de tipo {{domxref("BatteryManager")}}, está vacía y el sistema esta apunto de entrar en suspensión. Un valor de 1.0 significa que la `battery` está cargada al máximo. El valor 1.0 también nos indica que el sistema no puede determinar el nivel de carga de la batería o si no está alimentado por batería.
```

Comment:
Being a little picky again, mainly for consistency — locales often use `code` formatting for number values/measurements like these as well, though it's nothing major.

```suggestion
`Level` es un número que representa el nivel de carga de la batería del sistema en una escala entre `0.0` y `1.0`. Un valor de `0` significa que la `battery`, que es un objeto de tipo {{domxref("BatteryManager")}}, está vacía y el sistema esta apunto de entrar en suspensión. Un valor de `1.0` significa que la `battery` está cargada al máximo. El valor `1.0` también nos indica que el sistema no puede determinar el nivel de carga de la batería o si no está alimentado por batería.
```

Reply:
I haven't added them because in the EN file doesn't appear but yeah, I like it.